Dockerfile

```diff
@@ -29,8 +29,8 @@ ENV TERRA_APP_VERSION 0.5.0
 ENV GALAXY_VERSION 1.6.1
 ENV NGINX_VERSION 4.3.0
 # If you update this here, make sure to also update reference.conf:
-ENV CROMWELL_CHART_VERSION 0.2.187
-ENV CROWELL_ON_AZURE_CHART_VERSION 0.2.187
+ENV CROMWELL_CHART_VERSION 0.2.190
+ENV CROWELL_ON_AZURE_CHART_VERSION 0.2.190
```

Comment:
now that cromwhelm is updated, this should be `0.2.191` throughout

Reply:
I bumped it to 191 -- would rather just have 1 Leo PR